js/lib/scene.js

```diff
@@ -1,17 +1,16 @@
 define(['basic/entity', 'geo/v2', 'core/graphic', 'config/screen', 'config/config'],
 		function (Entity, V2, graphics, screen, config) {
-			function Scene() {
+		    function Scene() {
 				Entity.call(this, Zero(), new V2(screen.w, screen.h));
 				this.keyAware = [];
+			    this.bgColor = "#DDD";
 			}
 
 			Scene.prototype = new Entity();
 
 			Scene.prototype.onDraw = function (ctx) {
-				if (config.debug) {
-					ctx.fillStyle = "#dddddd";
-					ctx.fillRect(0, 0, this.size.x, this.size.y);
-				}
+				ctx.fillStyle = this.bgColor;
+				ctx.fillRect(0, 0, this.size.x, this.size.y);
```

Comment:
boah ey! scheiß doch auf die fillStyle color. dann halt nur das "ctx.fillRect(0, 0, this.size.x, this.size.y);" außerhalb der if! lass die zeile doch mal im if drin und mach config.debug auf false. das canvas gehält den alten zustand und überschreibt die geänderten objekte nicht richtig!!!!1elf